client/mksubvolume.cc

```diff
@@ -259,6 +278,104 @@ find_filesystem(MntTable& mnt_table)
 }
 
 
+string
+get_abs_subvol_path(string subvolume)
+{
+    if(!boost::starts_with(subvolume, "/"))
+	subvolume.insert(0, "/");
+    return subvolume;
+}
+
+
+void
+do_consistency_checks(MntTable& mnt_table, libmnt_fs* fs, libmnt_fs* expected_fs)
+{
+    char *subvol_expected, *subvol_fstab, *subvol_real;
+    const char *dev_expected, *dev_fstab;
```

Comment:
These variables should be declares as late as possible. Some are even only used in sub blocks.